nginx.conf

```diff
@@ -63,6 +66,8 @@ http {
         return 301 https://$host$request_uri;
       }
       <% end %>
+      add_header Cache-Control "no-cache";
+      add_header Pragma "no-cache";
```

Comment:
This block is for the static content served by analytics.usa.gov. I'm not sure setting the `Cache-Control` and `Pragma` headers to `no-cache` here makes sense, since this content is static and will not contain any private or sensitive information that we need to worry about caching.

I think we'd be better off setting this to `max-age=60` or something similar.